### ISSUE TITLE
Loss converged to negative values

### ISSUE THREAD

**Issue by sfchng** [Junyi42/monst3r#40]
Hi @Junyi42 

When I fine-tuned the model using the Sintel dataset, I noticed that the training loss eventually converged to negative values. Are you observing that as well, and why is that happening?

**Comment by Junyi42** [comment]
Hi @sfchng,

Yes, it is possible that the training loss converges to negative values. You could refer to eq(4) of the [DUSt3R paper](https://arxiv.org/abs/2312.14132) for more info.

<img width="430" alt="image" src="https://github.com/user-attachments/assets/cd19e594-a40b-4839-a326-0f133c3cc99a">


Best,
Junyi
